src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/coreperson/CorePersonMappingDto.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.coreperson
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Mappings for a Core Person and there associated child entities")
```

Comment:
```suggestion
@Schema(description = "Mappings for a Core Person and the associated child entities")
```

Reply:
Can't believe it says there and not their